driver/syscall_table.c

```diff
@@ -57,1937 +57,1267 @@ or GPL2.txt for full copies of the license.
  */
 const struct syscall_evt_pair g_syscall_table[SYSCALL_TABLE_SIZE] = {
 #ifdef __NR_open
-	[__NR_open - SYSCALL_TABLE_ID0] =                       {UF_USED | UF_NEVER_DROP, PPME_SYSCALL_OPEN_E, PPME_SYSCALL_OPEN_X},
+	[__NR_open - SYSCALL_TABLE_ID0] =                       {UF_USED | UF_NEVER_DROP, PPME_SYSCALL_OPEN_E, PPME_SYSCALL_OPEN_X, PPM_SC_OPEN},
```

Comment:
My rule of thumb to review this is:
* is github `files changed` gui only showing bold green highlight for the ppm_sc part? If yes, we are ok!